src/components/mortgage-investment-compare/input-card.js

```diff
@@ -38,6 +38,7 @@ const InputCard = props => {
               <InputGroupText>$</InputGroupText>
             </InputGroupAddon>
             <ValidatedNumberInput
+              max={500000000}
```

Comment:
hum..... but what if I'm buying the Buckingham Palace....? https://www.investopedia.com/articles/investing/060915/worlds-most-expensive-mansions.asp